cmd/proc_posix.go

```diff
@@ -22,9 +22,14 @@ import (
 )
 
 func gracefullyStopProcess(pid int) error {
+	fmt.Printf("Graceful stop...")
 	err := syscall.Kill(pid, syscall.SIGINT)
 	if err != nil {
 		return fmt.Errorf("kill: %v", err)
 	}
 	return nil
 }
+
+func getAppName() string {
```

Comment:
`getProcessName()` will be less ambiguous, since Caddy has "apps" that it runs internally.

Reply:
Thats fine, I was trying to somehow differentiate between the processes in the process list and the name of the .exe (or process), but I think getProcessName should be fine